habitat/core/dataset.py

```diff
@@ -89,6 +90,16 @@ def get_episodes(self, indexes: List[int]) -> List[T]:
         """
         return [self.episodes[episode_id] for episode_id in indexes]
 
+    def get_episode_iterator(self):
+        r"""
+        Creates and returns an iterator that iterates through self.episodes
+        in the desirable way specified.
```

Comment:
What are the ways?

Reply:
Basically there are two demanded features: one is shuffling before looping back after all episodes have been exhausted. The other one is switching scenes when episodes from the same scene is loaded too many times on a simulator, to prevent overfitting to that scene. Those features will be supported very soon (but maybe on another PR after this high-level change has been approved). For now the iterator is a simple infinite-loop over episodes.